sentry-sidekiq/spec/sentry/sidekiq/util_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe Sentry::Sidekiq::Util do
+  describe '.transaction_name_from_context' do
+    subject(:transaction_name) { described_class.transaction_name_from_context(context) }
+
+    context 'when the context is a job' do
+      let(:context) { { 'class' => 'FooJob' } }
+
+      it 'extracts the class' do
+        expect(transaction_name).to eq('Sidekiq/FooJob')
+      end
+    end
+
+    context 'when the context is a wrapped job' do
+      let(:context) { { 'wrapped' => 'FooJob', 'class' => 'WrapperJob' } }
+
+      it 'extracts the class' do
+        expect(transaction_name).to eq('Sidekiq/FooJob')
+      end
+    end
+
+    context 'when the context is a nested job' do
+      let(:context) { { job: { 'class' => 'FooJob' } } }
+
+      it 'extracts the class' do
+        expect(transaction_name).to eq('Sidekiq/FooJob')
+      end
+    end
+
+    context 'when the context is a wrapped nested job' do
+      let(:context) { { job: { 'wrapped' => 'FooJob', 'class' => 'WrapperJob' } } }
+
+      it 'extracts the class' do
+        expect(transaction_name).to eq('Sidekiq/FooJob')
+      end
+    end
+
+    context 'when the context is an event' do
+      let(:context) { { event: 'startup' } }
```

Comment:
I'm not too familiar with sidekiq, can `event` and other attributes like `wrapped`, `class` appears at the same time?

Reply:
My understanding, based on the [Sidekiq `fire_event` source](https://github.com/mperham/sidekiq/blob/02d265aac059644d42550677fdea77a7874f27c4/lib/sidekiq/util.rb#L52-L65), is that these 2 things cannot coexist.